services/web/server/src/simcore_service_webserver/folders/_folders_db.py

```diff
@@ -167,9 +173,9 @@ async def get(
         )
     )
 
-    async with get_database_engine(app).acquire() as conn:
+    async with pass_or_acquire_connection(get_asyncpg_engine(app), connection) as conn:
         result = await conn.execute(query)
-        row = await result.first()
+        row = result.first()
```

Comment:
Here as well. I would expect this one to be await maybe the `execute` above should be a `stream`?